stylesheets/_alerts.scss

```diff
@@ -1,49 +1,98 @@
-.alert {
-  color: #363636;
+div.alert-box {
+  color: #363636 !important;
   border-width: 1px;
   border-style: solid;
+  padding: 10px 20px;
+  margin: 1.5em auto;
+  font-size: 1em;
+
+  div.alert-content {
+    display: inline;
+    strong { margin-right: .5em; }
+    p { 
+      margin-bottom: 0;
+      display: inline;
+    }
+  }
+
+  div.icon {
+    width: 1.5em;
+    height: 1.5em;
+    display: inline-block;
+    position: relative;
+    margin-right: 10px;
+    top: 6px;
+  }
 
   &.alert-info {
     background-color: #dcedf8;
     border-color: #87aac1;
 
-    &.icon {
-      background-image: cdn("../images/icons/alerts/RHD_alerticon_info.png");
+    div.icon {
+      &:before { content: url('http://static.jboss.org/rhd/images/icons/RHD_alerticon_info.svg'); }
```

Comment:
@adelasofia would be better to use either https or just use //static... to avoid "mixed content" warning

Reply:
Sorry about that, I didn't realize I was using http. I'm raising a PR to fix this 😄